integration/docker_engine_test.go

```diff
@@ -226,11 +226,17 @@ func TestRun_WithDockerEngine(t *testing.T) {
 			expectedOutputFile: "testdata/photon-30.json.golden",
 			testfile:           "testdata/fixtures/photon-30.tar.gz",
 		},
+		{
+			name:               "buxybox with Cargo.lock integration",
+			imageTag:           "busy-cargo:latest",
+			expectedOutputFile: "testdata/busybox-with-lockfile.json.golden",
+			testfile:           "testdata/fixtures/busybox-with-lockfile.tar.gz",
+		},
 		{
 			name:          "sad path, invalid image",
 			invalidImage:  true,
 			testfile:      "badimage:latest",
-			expectedError: "unable to access the remote image (index.docker.io/library/badimage:latest)",
+			expectedError: "unable to inspect the image (index.docker.io/library/badimage:latest): Error: No such image: index.docker.io/library/badimage:latest",
```

Comment:
@knqyf263 I wasn't able to figure out why this error has changed. Do you know?

Reply:
Yes, you can find it here.
https://github.com/aquasecurity/fanal/pull/101/files#diff-525dd35201b72b43ad63b85dd2720ba6R66-R87